src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -215,9 +214,7 @@ private function getConfiguration()
         if (null !== $this->configuration) {
             return $this->configuration;
         }
-        if (false === $mode = $this->mode) {
```

Comment:
Not sure this is legit: the property is used L88.

Reply:
Oh, you're right, good catch! Closing then.